src/domain/checkForAndPlaceOrder.ts

```diff
@@ -550,22 +549,29 @@ async function _pollLegacy(
       signature,
     };
   } catch (error: any) {
-    log.error(
```

Comment:
not from this PR, but, i think the `low-level call failed` log message is insufficient. It should be giving a context the call was `getTradeableOrderWithSignature`

Reply:
okokko